ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/BoardPanel.java

```diff
@@ -37,16 +37,24 @@ public BoardPanel(BoardSize boardSize, SquareConfig squareConfig,
         mBoardSize = boardSize;
         mSquareLabels = new ArrayList<>(boardSize.width * boardSize.height);
 
-        GridLayout gridLayout = new GridLayout(mBoardSize.width + 2, mBoardSize.height + 2);
+        GridLayout gridLayout = new GridLayout(mBoardSize.height + 1, mBoardSize.width + 2);
```

Comment:
Also, we could have a second `BoardSize` instance variable, `mTotalBoardSize = BoardSize.withDimensions(mBoardSize + HORIZONTAL_PADDING, mBoardSize.height + VERTICAL_PADDING)` and that should make the math simpler